upgrade_test.py

```diff
@@ -762,6 +762,107 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
 
         InfoEvent(message='all nodes were upgraded, and last workaround is verified.').publish()
 
+    def test_custom_profile_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
+        """
+        Run a load of a custom profile.
```

Comment:
Is this whole flow copied from a different test method ? Maybe we should consider sharing most of it ?